### ISSUE TITLE
"lang" attribute not being rendered in language dropdown menu.

### ISSUE THREAD

**Issue by michael-milette** [trema-tech/moodle-theme_trema#58]
Each of the languages should be spoken by screen readers in their own language and therefore should have a lang="en" or lang="xx".

![image](https://user-images.githubusercontent.com/3808579/138205440-7f8b4238-d59c-4296-8cf9-de7f23a529fe.png)

I can see in your code that you are attempting to add it in classes/output/core_renderer.php line 104 but for some reason it isn't being rendered as expected.

    $this->language->add($langname, new moodle_url($this->page->url, array('lang' => $langtype)), $langname);

If you have any ideas, let me know. Otherwise I hope to get around to looking into this bug within the next week.

Best regsards,

Michael Milette

**Comment by michael-milette** [comment]
In case you are wondering how Boost handles this, it has the same problem however it is coded differently so the same solution cannot applied in core. For more information, see https://tracker.moodle.org/browse/MDL-72885
